src/app/_components/page-content/DownloadCard.tsx

```diff
@@ -115,7 +156,7 @@ export const DownloadCard = ({ data, rows }) =>
                     className='fr-p-3w'
                     style={{ width: '100%', objectFit: 'contain' }}
                     src={data.image_de_la_carte.data.attributes.url}
-                    alt=''
+                    alt='Illustration'
```

Comment:
```suggestion
                    alt=''
```
`Illustration` n'apporte pas d'information essentielle sur le contenu à une personne utilisant des outils d'accessibilité